mainwindow.h

```diff
@@ -23,19 +23,20 @@
 
 enum WorkingModes
 {
-    MANUAL_MODE = 0,
+// QA: why you declare MANUAL_MODE, it will be the same 
+// QA: if you won't declare it, just MANUL_MODE, ...
+    MANUAL_MODE = 0, 
```

Comment:
I'm very suspicious person, so I just wanted to ensure...

Reply:
only for MANUAL_MODE? What about others? Do you trust them?